airflow/providers/google/cloud/transfers/bigquery_to_mssql.py

```diff
@@ -18,19 +18,18 @@
 """This module contains Google BigQuery to MSSQL operator."""
 from __future__ import annotations
 
+import warnings
 from typing import TYPE_CHECKING, Sequence
 
-from airflow.models import BaseOperator
-from airflow.providers.google.cloud.hooks.bigquery import BigQueryHook
 from airflow.providers.google.cloud.links.bigquery import BigQueryTableLink
-from airflow.providers.google.cloud.utils.bigquery_get_data import bigquery_get_data
+from airflow.providers.google.cloud.transfers.bigquery_to_sql import BigQueryToSqlBaseOperator
 from airflow.providers.microsoft.mssql.hooks.mssql import MsSqlHook
 
 if TYPE_CHECKING:
     from airflow.utils.context import Context
 
 
-class BigQueryToMsSqlOperator(BaseOperator):
+class BigQueryToMsSqlOperator(BigQueryToSqlBaseOperator):
```

Comment:
This class is missing from the added docs. is this intentional?

Reply:
I was inspired by the implementation of `BaseSQLToGCSOperator`. This is also an abstract class which is not documented either.

Reply:
I'm asking about the `BigQueryToMsSqlOperator` not on the base class :)

Reply:
Yes it is intentional. There was a slight difference between the MySQL documentation and the MsSQL documentation. I standardized the location of the example snippet. I used the MsSQL version. It did not require to change the MsSQL version, but this is why there is a change on the MySQL version.

As for the new interface (`s/mssql_table/target_table_name/`), it has been changed directly in `example_bigquery_to_mssql.py`.

Reply:
I'm asking about the rst files.

docs/apache-airflow-providers-google/operators/transfer/bigquery_to_mssql.rst
Is missing.

Reply:
@jbbqqf can you add `docs/apache-airflow-providers-google/operators/transfer/bigquery_to_mssql.rst`
 ?